css/src/components/form/form.scss

```diff
@@ -0,0 +1,241 @@
+/* stylelint-disable max-nesting-depth, selector-max-specificity  */
+
+$form-font-size: $font-size-7;
+$form-font-size-sm: $font-size-8;
+$form-font-size-lg: $font-size-6;
+
+$icon-font-size: $font-size-7;
+$icon-font-size-sm: $font-size-8;
+$icon-font-size-lg: $font-size-6;
+$icon-size: 2.25em;
+
+$icon-color: $secondary-background !default;
+$icon-active-color: $secondary !default;
+
+.control {
+	position: relative;
+	font-size: $form-font-size;
+	text-align: left;
+
+	&.control-icons {
+		.icon {
+			position: absolute;
+			inset-block-start: 0;
+			inset-inline-start: 0;
+			width: $icon-size;
+			height: $icon-size;
+			color: $icon-color;
+			font-size: $icon-font-size;
+			pointer-events: none;
+			z-index: $zindex-multi;
+			padding-inline-start: $icon-size;
+		}
+
+		.input,
+		.select select {
+			padding-inline-start: $icon-size;
+		}
+
+		.input,
+		.select {
+			~ .icon {
+				font-size: $icon-font-size;
+			}
+
+			&:focus-visible {
+				~ .icon,
+				.icon {
+					color: $icon-active-color;
+				}
+			}
+
+			&.input-sm,
+			&.select-sm {
+				~ .icon,
+				.icon {
+					font-size: $icon-font-size-sm;
+				}
+			}
+
+			&.input-lg,
+			&.select-lg {
+				~ .icon,
+				.icon {
+					font-size: $icon-font-size-lg;
+				}
+			}
+		}
+
+		&.control-icon-right {
+			.input {
+				padding-inline: $control-padding-horizontal;
+
+				~ .icon {
+					inset-inline-start: unset;
+					inset-inline-end: 0;
+				}
+			}
+
+			.select {
+				~ .icon {
+					inset-inline-start: unset;
+					inset-inline-end: 2.25em;
+				}
+			}
+
+			.input,
+			.select select {
+				padding-inline-end: $icon-size;
+			}
+
+			.input[type='date'] {
+				position: relative;
+
+				&::-webkit-calendar-picker-indicator {
```

Comment:
This indicator is used here https://review.docs.microsoft.com/en-us/test/learn/certifications/courses/az-203t01?branch=main&preview=ilt